clouddriver-sql/src/main/resources/db/changelog/20181120-cats.yml

```diff
@@ -216,6 +216,7 @@ databaseChangeLog:
         newDataType: varchar(255)
 
 - changeSet:
+    validCheckSum: 8:e252377f81b5153444f82dd257b1db7a
```

Comment:
Can you explain where this value (and the one below) come from?

Reply:
This checksum value is corresponding to the changeSet - `20190913-task-sagaids.yml::add-task-sagaids-column` and liquibase 4.27.0 combination. i.e, for the same changeSet and liquibase 4.27.0, we get the same checksum - 8:e252377f81b5153444f82dd257b1db7a in any machine.

So when migrating from liquibase 3.10.0 to 4.27.0 (spinnaker <1.33.0 to Spinnaker >=1.33.0) I received this error:
 `...db/changelog/20181120-cats.yml::20181120-cats.yml::application-index::afeldman was: 8:03b00d0af09f2e7081d187f246ea4d26 but is now: 8:e252377f81b5153444f82dd257b1db7a`

 Similarly for the other changeset I got this - `db/changelog/20190913-task-sagaids.yml::add-task-sagaids-column::robzienert was: 8:91cfabe4a8fa0517124436ef9675708e but is now: 8:dac7716de0de84f4fd508d906d13bcaa`